### ISSUE TITLE
android/386 android/amd64

### ISSUE THREAD

**Issue by dzeckelev** [gomatcha/matcha#52]
Hi. Please tell why they removed these lines from the file?
I could not find the answer to this question? sorry if already answered it.
in `matcha/cmd/bind.go`

> //targets["android/386"] = struct{}{}
> //targets["android/amd64"] = struct{}{}

**Comment by overcyn** [comment]
What do you need it for? I’m not too knowledgeable about android development.

**Comment by dzeckelev** [comment]
it is convenient to debug the code using the Genymotion emulator for Android.

Genymotion launches x86 android images.
Many devices on the platform Intel Atom x86 and Atom x64.
P.S. the question is not the topic. I still could not launch the  Insta example in Android. How can I track errors through the Matcha?

**Comment by overcyn** [comment]
Readded the targets here https://github.com/gomatcha/matcha/pull/53.

**Comment by overcyn** [comment]
What error do you get when you launch the Insta example?

**Comment by dzeckelev** [comment]
> What error do you get when you launch the Insta example?

Android Studio 3.0
Android 7.1.1 (API 25)
Go 1.9.1
Matcha Latast

[simple_app.txt](https://github.com/gomatcha/matcha/files/1442876/insta_app.txt)

**Comment by overcyn** [comment]
Weird, does it crash consistently? I've been trying to track down a similar crash that happens very occasionally for me. I think there is a bug in the Android to Go bridge code.

**Comment by dzeckelev** [comment]
> Weird, does it crash consistently? I've been trying to track down a similar crash that happens very occasionally for me. I think there is a bug in the Android to Go bridge code.
Yes, the application always crashes. This bug is always reproduced on android.

**Comment by overcyn** [comment]
For some context, the crash is at Java-Tracker.java:64. From Go we try to call a method on a java object using reflection. But the java object returned by `this.get(v)` is not the one we expect, which leads to InvocationTargetException. Why the tracker has the wrong object or Go has the wrong foreignId is unknown.

**Comment by overcyn** [comment]
Cleaned up the JNI code a bit here https://github.com/gomatcha/matcha/pull/54. Not sure if it would fix your crash but might be worth trying.

**Comment by dzeckelev** [comment]
The error is the same. Resource ID #0x0 . So it should be? 

```
11-10 05:45:05.658 1999-1999/com.example.bik.insta I/InstantRun: starting instant run server: is main process
11-10 05:45:05.756 1999-1999/com.example.bik.insta W/art: Before Android 4.1, method android.graphics.PorterDuffColorFilter android.support.graphics.drawable.VectorDrawableCompat.updateTintFilter(android.graphics.PorterDuffColorFilter, android.content.res.ColorStateList, android.graphics.PorterDuff$Mode) would have incorrectly overridden the package-private method in android.graphics.drawable.Drawable
                                                          
                                                          [ 11-10 05:45:06.050  1999: 1999 D/         ]
                                                          HostConnection::get() New Host Connection established 0xdc184880, tid 1999
                                                          
                                                          
                                                          [ 11-10 05:45:06.052  1999: 1999 W/         ]
                                                          Process pipe failed
11-10 05:45:06.142 1999-1999/com.example.bik.insta W/ResourceType: No package identifier when getting value for resource number 0x00000000
11-10 05:45:06.143 1999-1999/com.example.bik.insta V/Bridge: foreignCall, 1, getPropertiesForResource, [2], android.content.res.Resources$NotFoundException: Resource ID #0x0
```
